app/views/devise/mailer/email_changed.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @email %>!</p>
```

Comment:
This is out of scope for this PR. We might want to use the greeting name like done [here](https://github.com/pieforproviders/pieforproviders/pull/208/files#diff-6909b6f56fb271baaecf5e7ef68425d9R3).

Reply:
Ya these emails haven't been designed, not sure they're even on the design radar, we should ping @csprayregen about generating tickets for these emails.

Reply:
PS. Chelsea, all the mailers that AREN'T confirmation_instructions don't need translation; we don't have a design/layout for them yet I don't think; that's what the tickets should be.